cellprofiler/modules/plugins/measurementtemplate.py

```diff
@@ -64,14 +152,14 @@
 #
 # The module class
 #
-# Your module should "inherit" from cellprofiler.cpmodule.CPModule.
-# This means that your module will use the methods from CPModule unless
-# you re-implement them. You can let CPModule do most of the work and
+# Your module should "inherit" from cellprofiler.module.Module.
+# This means that your module will use the methods from Module unless
+# you re-implement them. You can let Module do most of the work and
 # implement only what you need.
 #
 ###################################
 
-class MeasurementTemplate(cpm.Module):
+class MeasurementTemplate(cellprofiler.module.Module):
```

Comment:
Would it make sense to have a `MeasurementProcessing` class to inherit from?

Reply:
Hmm... I'm not sure. We'd need to define the commonalities between measurement modules (beyond "they make measurements"). Currently, I'm not sure what abstractions would be useful. I'm open to ideas.

Reply:
I haven't written any measurement modules, so I'm not sure what could be refactored either :) I'll keep an open mind if I work on them in the future!